### ISSUE TITLE
pip intall error

### ISSUE THREAD

**Issue by alex-maxime** [mapzen/tileserver#117]
Hello team :)

I have an error when i am trying to install tile server,
```shell
pip install -Ur tileserver/requirements.txt
Collecting AppDirs==1.4.3 (from -r tileserver/requirements.txt (line 1))
  Using cached appdirs-1.4.3-py2.py3-none-any.whl
Requirement already up-to-date: argparse==1.2.1 in /usr/lib/python2.7 (from -r tileserver/requirements.txt (line 2))
[...]
  Could not find a version that satisfies the requirement raw_tiles (from tilequeue->-r tileserver/requirements.txt (line 27)) (from versions: )
No matching distribution found for raw_tiles (from tilequeue->-r tileserver/requirements.txt (line 27))
```

can you help me to resolve this ?.

Thanks to you :)

**Comment by rmarianski** [comment]
Whoops, we were missing a requirement. I just pushed over a fix with bdc6ac0. Can you try again?

**Comment by alex-maxime** [comment]
Good, it work now, thank you
